distributed-databases/trino/main.tf

```diff
@@ -0,0 +1,114 @@
+module "eks" {
```

Comment:
Please change the name of this file to `eks.tf`. Then, transfer the code of `locals.tf` and `data.tf` into `main.tf`. Once you've moved everything over, you can go ahead and delete the locals.tf and data.tf files.

Reply:
addressed